source/server/http/admin.cc

```diff
@@ -742,7 +742,7 @@ uint64_t PrometheusStatsFormatter::statsAsPrometheus(
     const std::vector<double>& computed_buckets = stats.computedBuckets();
     for (size_t i = 0; i < supported_buckets.size(); ++i) {
       double bucket = supported_buckets[i];
-      double value = computed_buckets[i];
+      int value = computed_buckets[i];
```

Comment:
Please use uint64_t here.

Also, would it make more sense to store stats.computedBuckets() as uint64_t instead of double? I normally think of histograms in terms of integer counts instead of real numbers.